index.html

```diff
@@ -86,116 +86,132 @@ <h2 id="position">I’m a software developer</h2>
           </ul>
         </div>
         <div class="scroll">
-          <img src="Icons/scroll_btn.png" alt="scroll down button" />
+          <a href="#works_main">
+            <img
+              src="Icons/scroll_btn.png"
+              alt="scroll down button"
+              id="scroll_image"
+            />
+          </a>
         </div>
       </div>
     </section>
-    <section class="works_main">
+    <section class="works_main" id="works_main">
       <div class="works_heading">
         <h1 class="heading">My Recent Works</h1>
```

Comment:
- [OPTIONAL]  I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

> You're used it once here ` <h1 id="name">Hey there. I’m Yonas</h1>` and It's perfect because this is the main title 💯

Reply:
Okay thanks I will have them fixed!